.github/workflows/ci.yml

```diff
@@ -110,6 +93,26 @@ jobs:
         run: yarn lerna run test-unit-ibmi --scope=@sequelize/core
       - name: Unit tests (snowflake)
         run: yarn lerna run test-unit-snowflake --scope=@sequelize/core
+  unit-test-win:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16]
+    name: Build and test on Windows (Node ${{ matrix.node-version }})
+    runs-on: lint
```

Comment:
This does not work, `lint` is not a possible OS

Reply:
Ahh, it seems I did fix that, just on the wrong branch... Let me merge that change in...